### PR TITLE
add dygraph+parl2.0+paddle2.0 version of `tutorials` code for bilibili course

### PR DESCRIPTION
**Pull request by rical730**
lesson3
lesson4
lesson5

### CODE REVIEW

File: examples/tutorials/README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ## 《PARL强化学习入门实践》课程示例
 
-针对强化学习初学者，PARL提供了[入门课程](https://aistudio.baidu.com/aistudio/course/introduce/1335)，展示最基础的5个强化学习算法代码示例（注意：本课程示例均基于静态图框架编写）。
+针对强化学习初学者，PARL提供了[入门课程](https://aistudio.baidu.com/aistudio/course/introduce/1335)，展示最基础的5个强化学习算法代码示例（注意：本课程示例均基于**静态图框架**编写）。
+
+## News: 
++ 应广大学员的要求，我们提供了课程配套代码的（lesson3-lesson5）的**动态图框架**版本，见 [链接](parl2_dygraph/)。
```
Comment (TomorrowIsAnOtherDay) [inline examples/tutorials/README.md]: Please add time information

File: examples/tutorials/parl2_dygraph/lesson3/dqn/algorithm.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        # 获取Q预测值
+        pred_values = self.model(obs)
+        action_dim = pred_values.shape[-1]
+        action = paddle.squeeze(action, axis=-1)
+        # 将action转onehot向量，比如：3 => [0,0,0,1,0]
+        action_onehot = paddle.nn.functional.one_hot(
+            action, num_classes=action_dim)
+        # 下面一行是逐元素相乘，拿到action对应的 Q(s,a)
+        # 比如：pred_value = [[2.3, 5.7, 1.2, 3.9, 1.4]], action_onehot = [[0,0,0,1,0]]
+        pred_value = paddle.multiply(pred_values, action_onehot)
```
Comment (TomorrowIsAnOtherDay) [inline examples/tutorials/parl2_dygraph/lesson3/dqn/algorithm.py]: why not just use * here?

**Review by TomorrowIsAnOtherDay — COMMENTED** [review]
Please check if the code can run in the GPU environment with the GPU being used correctly.

File: examples/tutorials/parl2_dygraph/lesson3/dqn/algorithm.py (hunk 2 above)
Comment (rical730) [reply examples/tutorials/parl2_dygraph/lesson3/dqn/algorithm.py]: done

File: examples/tutorials/README.md (hunk 1 above)
Comment (rical730) [reply examples/tutorials/README.md]: done

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
LGTM.

**Pull request merged**